actions/handle-transaction.ts

```diff
@@ -76,3 +76,21 @@ export async function insertTransactions(formData: NewTransactionSchemaType) {
     return { error: "An error occured while adding new transaction" };
   }
 }
+
+export async function deleteTransaction(transactionId: string) {
+  try {
+    const { transaction, error } = await getTransactionById(transactionId);
+
+    if (error || !transaction || transaction.length === 0)
```

Comment:
Return the `error` as it is from `getTransactionById`

Reply:
Hmmm... I see...

Reply:
Done it.

Reply:
Dont it